src/content/fr/2019/fonts.md

```diff
@@ -0,0 +1,682 @@
+  caption="Distribution des requêtes de police par page.",
+  description="Graphique à barres indiquant la répartition des demandes de polices par page. Les 10, 25, 50, 75 et 90e percentiles pour le bureau sont : 0, 1, 3, 6 et 9 demandes de polices. La distribution pour les mobiles est identique jusqu'aux 75e et 90e percentiles, où les pages mobiles demandent une police de moins.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQDogXDb3BwZZHrBT39qccP_LJoCScD3QEi_FmjT_8VDPD_1Srpz-g7ZuuTUEb8pYXBpDmQzZ1hQh7q/pubchart?oid=451821825&format=interactive"
+  )
+}}
+
+La page web médiane fait trois demandes de polices de caractères web. Au 90e percentile, elle a demandé six et neuf polices web sur le mobile et le bureau, respectivement.
+
+{{ figure_markup(
+  image="fig13.png",
+  caption="Histogram of web fonts requested per page.",
+  description="Histogramme montrant la répartition du nombre de demandes de polices par page. Le nombre de demandes de polices le plus répandu est 0 pour 22 % des pages pour ordinateur de bureau. La distribution tombe à 9 % des pages ayant une police, puis atteint un maximum de 10 % pour 2 à 4 polices avant de diminuer à mesure que le nombre de polices augmente. Les distributions pour les ordinateurs de bureau et les téléphones mobiles sont similaires, bien que la distribution pour les téléphones mobiles penche légèrement vers une réduction du nombre de polices par page.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQDogXDb3BwZZHrBT39qccP_LJoCScD3QEi_FmjT_8VDPD_1Srpz-g7ZuuTUEb8pYXBpDmQzZ1hQh7q/pubchart?oid=1755200484&format=interactive"
+  )
+}}
+
+Il est intéressant de constater que les demandes de polices de caractères sur le web semblent être assez stables sur les ordinateurs de bureau et les téléphones portables. Je suis heureux de constater que la <a hreflang="en" href="https://css-tricks.com/snippets/css/using-font-face/#article-header-id-6">recommandation de cacher les blocs `@font-face` dans les requêtes `@media`</a> n'a pas été retenue (ne vous faites pas d'idées).
+
+Cela dit, les requêtes de polices de caractères faites sur les appareils mobiles sont légèrement plus nombreuses. J'ai l'impression que moins de polices sont disponibles sur les appareils mobiles, ce qui signifie moins d'occurrences `local()` dans les CSS de Google Fonts, et donc moins de requêtes sur le réseau pour ces polices.
+
+### Vous ne voulez pas gagner ce prix
+
+{{ figure_markup(
+  caption="Le plus grand nombre de demandes de polices Web sur une seule page.",
+  content="718",
+  classes="big-number"
+)
+}}
+
+Le prix de la page qui demande le plus de polices web est attribué à un site qui a fait **718** requêtes de polices web !
+
+Après avoir plongé dans le code, toutes ces 718 requêtes vont à Google Fonts ! Il semble qu'un plugin d'optimisation "Above the Page fold" pour WordPress qui ne fonctionne pas bien soit devenu défectueux sur ce site et demande (DDoS-ing ?) toutes les polices Google !
```

Comment:
Chevrons et em lang="en" pour "Above the Page fold"

Reply:
Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/commit/6df484a83f05c285020d15faa5a0207a42c519e7